webapp/src/js/services/android-api.js

```diff
@@ -171,6 +171,18 @@ angular.module('inboxServices').factory('AndroidApi',
           }
         },
 
+        /**
+         * Handle the response from the MRDT app
+         * @param response The stringified JSON response from the MRDT app.
+         */
+        mrdtTimeTakenResponse: function(response) {
+          try {
+            MRDT.respondTimeTaken(JSON.parse(response));
```

Comment:
I'm a little confused by this `JSON.parse` call ... sorry, I haven't just set this up and installed the new app to see what the data looks like.  What exactly is in `response`? If it is an object, then [here](https://github.com/medic/medic/pull/5728/files#diff-dde8d896f9bfeefb03f1d8bfc817fca1R37) `timeTaken` would be an object, and then [here](https://github.com/medic/medic/pull/5728/files#diff-a8375f122d90b219ecbab54fb58a8154R74) the value being put into the input would be an object.  Am I missing something?

Reply:
It is a JSON string that is passed .. then we do something like `JSON.parse('"kenn"');` wich produces the string `kenn`. I can't think of a reason why it was setup this way instead of passing a base 64 string directly. Any idea?

Reply:
```
JSON.parse("kenn")
VM62:1 Uncaught SyntaxError: Unexpected token k in JSON at position 0
```

?

Reply:
you forgot the single quotes `'"kenn"'`

Reply:
oh man... that's a super weird way to pass data :)